sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -903,11 +903,13 @@ typedef struct CommandListBatchConfig {
   bool dynamic() const { return Size == 0; }
 } zeCommandListBatchConfig;
 
-static const zeCommandListBatchConfig ZeCommandListBatch = [] {
+static const zeCommandListBatchConfig ZeCommandListBatchConfig(bool IsCopy) {
```

Comment:
This was a static variable with the goal to be initialized only once, and you made it into a function that will be executed every time you query config. Please turn this into 2 static variables initialized once (re-use initialization code).

Reply:
Done. Thanks. I agree. Multiple calls is more expensive here.